specs/tasks.js

```diff
@@ -143,13 +147,23 @@ function slashToUnderscore(string) {
 }
 
 function replace() {
-  return function(selector, content) {
-    document.querySelector(selector).innerHTML = content
+  return function(selector, content, isXPath) {
+    if (isXPath) {
+      let node = document.evaluate(selector, document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue
```

Comment:
As I see, we can replace only one node/element

It would be nice to cover cases when we want to make multiple replacements (fe. table with date column, we could replace all dates with one value). So option is to use `let xpathRes = document.evaluate('//*[@id="block-thunder-admin-content"]/div[3]/text()[1]', document);` and also `document.querySelectorAll('td').forEach(function(elem){elem.innerHTML='<td>test</td>';});` for CSS selectors.